packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -143,31 +142,39 @@ deployFunctionApp() {
     echo "Successfully deployed Azure Function App '$myFunctionAppName'"
 }
 
-function deployWebApiFunction {
+function deployWebApiFunction() {
     deployFunctionApp "web-api-allyfuncapp" "${0%/*}/../templates/function-web-api-app-template.json" "$webApiFuncAppName" "clientId=$webApiAdClientId"
 }
 
-function deployWebWorkersFunction {
+function deployWebWorkersFunction() {
     deployFunctionApp "web-workers-allyfuncapp" "${0%/*}/../templates/function-web-workers-app-template.json" "$webWorkersFuncAppName"
 }
 
-function enableManagedIdentityOnFunctions {
+function enableManagedIdentityOnFunctions() {
     getFunctionAppPrincipalId $webApiFuncAppName
     . "${0%/*}/key-vault-enable-msi.sh"
 
+    role="Storage Blob Data Contributor"
+    scope="--scope /subscriptions/$subscription/resourceGroups/$resourceGroupName/providers/Microsoft.Storage/storageAccounts/$storageAccountName"
+    . "${0%/*}/role-assign-for-sp.sh"
```

Comment:
Could we put this reused code (the first 5 lines of this func) in its own function that takes the function app name as an argument?